src/main/java/timify/com/subject/SubjectController.java

```diff
@@ -79,4 +78,26 @@ public ApiResponse<updateTitleNameDto> updateTitle(@RequestBody @Valid subjectRe
     return ApiResponse.of(SuccessStatus.TITLE_CHANGE_SUCCESS, updateTitleNameDto);
   }
 
+  @PutMapping("/{id}/store")
+  @Operation(summary = "항목 보관함 이동 API", description = "항목을 보관함으로 이동시키는 API 입니다.")
+  public ApiResponse<Long> storeSubject(@PathVariable Long id) {
```

Comment:
```suggestion
    @PostMapping("/type/{studyTypeId}/update")
    @Operation(summary = "공부 분류 수정 API", description = "특정 공부 분류의 이름을 수정하는 API 입니다.")
    @Parameters(value = {
            @Parameter(name = "studyTypeId", description = "공부 분류의 id 입니다.")
    })
    public ApiResponse<StudyResponse.studyTypeDto> updateStudyType(
```
path variable이 있는 경우에는 위 처럼 @Parameters를 이용해서 매핑을 해주어야 스웨거에서 테스트가 가능합니다..!

Reply:
확인하였습니다.